diff --git a/resources/taxonomy_resource.inc b/resources/taxonomy_resource.inc
index fcdd940..556215c 100644
--- a/resources/taxonomy_resource.inc
+++ b/resources/taxonomy_resource.inc
@@ -330,7 +330,11 @@ function _taxonomy_term_resource_create($term) {
   $term = _services_arg_value($term, 'term');
 
   $term = (object)$term;
-  return taxonomy_term_save($term);
+  if (taxonomy_term_save($term) == 1) {
+    return array("tid" => $term->tid);
+  } else {
+    return services_error(t('Unable to create new taxonomy term. Please contact administrator'), 404);
+  }
 }
 
 /**
@@ -620,4 +624,4 @@ function _taxonomy_vocabulary_resource_index($page, $fields, $parameters, $page_
   $results = $taxonomy_select->execute();
 
   return services_resource_build_index_list($results, 'taxonomy_vocabulary', 'vid');
}
