diff --git a/resources/taxonomy_resource.inc b/resources/taxonomy_resource.inc
index 074f791..7721756 100644
--- a/resources/taxonomy_resource.inc
+++ b/resources/taxonomy_resource.inc
@@ -393,7 +393,11 @@ function _taxonomy_term_resource_create($term) {
   $term = _services_arg_value($term, 'term');
 
   $term = (object)$term;
-  return taxonomy_term_save($term);
+  if (taxonomy_term_save($term) == 1) {
+    return array("tid" => $term->tid);
+  } else {
+    return services_error(t('Unable to create new taxonomy term. Please contact administrator'), 404);
+  }
 }
 
 /**
