Index: auto_nodetitle.module
===================================================================
--- auto_nodetitle.module	(revision 886)
+++ auto_nodetitle.module	(working copy)
@@ -128,7 +128,7 @@
   */
 function _auto_nodetitle_patternprocessor($pattern, $node) {
   // Replace tokens.
-  $output = token_replace($pattern, array('node' => $node), array('sanitize' => FALSE));
+  $output = token_replace($pattern, array('node' => $node), array('callback' => '_auto_nodetitle_nohtmlentities', 'sanitize' => FALSE, 'clear' => TRUE));
   // Evalute PHP.
   if (variable_get('ant_php_' . $node->type, 0)) {
     $output = auto_nodetitle_eval($output, $node);
@@ -139,6 +139,15 @@
 }
 
 /**
+* Callback function to remove entities.
+*/
+function _auto_nodetitle_nohtmlentities(&$replacements, $data, $options) {
+  foreach ($replacements as $key => $value) {
+    $replacements[$key] = decode_entities($value);
+  }
+}
+
+/**
  * Implements hook_form_FORM_ID_alter() for the node type form.
  */
 function auto_nodetitle_form_node_type_form_alter(&$form, &$form_state) {
